packages/swapkit/swapkit-core/src/aggregator/getSwapInParams.ts

```diff
@@ -36,19 +38,24 @@ export const getSwapInParams = ({
    * GENERIC: tcRouter, tcVault, tcMemo, token, amount, router, data, deadline
    * ETH_UNISWAP: tcRouter, tcVault, tcMemo, token, amount, amountOutMin, deadline
    * AVAX_PANGOLIN: tcRouter, tcVault, tcMemo, token, amount, amountOutMin, deadline
-   * AVAX_WOOFI: tcRouter, tcVault, tcMemo, token, amount, amountOutMin, deadline
+   * AVAX_WOOFI: router, vault, memo, token, amount, amountOutMin, deadline
    */
 
+  if (!tcVault && !vault) throw new Error('TC Vault is required on calldata');
+  if (!tcRouter && !router) throw new Error('TC Router is required on calldata');
```

Comment:
Should add validation for !tcMemo && !memo

We will always have a memo for swapIns